Dockerfile

```diff
@@ -28,17 +28,20 @@ RUN conda config --add channels conda-forge && conda config --add channels defau
   conda update -n base -c defaults conda
 
 # Create environment
-RUN conda create -n mssenv python=3
+RUN conda create -n mssenv python=3 \
```

Comment:
on this creation we can also directly add mamba on it

RUN conda create -n mssenv python=3 mamba